k8_cortx_cloud/cortx-cloud-helm-pkg/cortx-control/templates/cortx-control-pod.yaml

```diff
@@ -200,7 +99,8 @@ spec:
             - /bin/sh
           args:
             - -c
-            - /opt/seagate/cortx/csm/bin/csm_agent start -c yaml:///etc/cortx/cluster.conf
+            - /opt/seagate/cortx/csm/bin/csm_agent start -c yaml:///etc/cortx/cluster.conf;
```

Comment:
If the start command fails, should the container sleep forever, or should it exit for k8s to handle?

Reply:
This is a good question. We'll want to raise this in a larger concern in the format of these entrypoints. But for now, I would expect we want the container to stay alive to debug instead of crash-looping.

Reply:
Gaining the ability to debug the Pod is useful, but then do you lose the ability to (easily) know that anything is wrong in the first place?